kensu/pandas/data_frame.py

```diff
@@ -212,6 +219,9 @@ def wrapper(*args, **kwargs):
                         for col in result:
                             if values is not None:
                                 for val in values:
+                                    if len(values) == 1:
+                                        kensu.add_dependencies_mapping(result_sc.to_guid(), str(col), orig_sc.to_guid(),
```

Comment:
P.S. this might sometimes add mapping twice (also in the line below as there's no `else`). is this ok?